### ISSUE TITLE
validate predictions

### ISSUE THREAD

**Issue by wconnell** [wconnell/enformer-finetune#10]
Include analyses to 

- [x] Inference on validation set and save predictions. We can do this by including a call to `trainer.predict()` and just pass the `val_dataloader`. We can then save the predictions (which should be in the same order as the `val_dataset`). Can do this in the script after the `.fit()` call or directly in an interactive env with the appropriate mem resources.
- [x] Plot correlations and retrieve statistics (compared to untrained model inference) [colab](https://colab.research.google.com/drive/1t135ZJF_9z1aDiUGzT1gV2zFdXlb2wTq#scrollTo=MpAC6lFNiA1g)
- [x] Plot some tracks and visually inspect [colab](https://colab.research.google.com/drive/1t135ZJF_9z1aDiUGzT1gV2zFdXlb2wTq#scrollTo=2OZRNN-rgLs7)

**Comment by aaronwtr** [comment]
started working on track visualisation on the [validate](https://github.com/wconnell/enformer-finetune/tree/validate) branch.
